web/src/attachments_ui.ts

```diff
@@ -196,6 +196,10 @@ export function set_up_attachments(): void {
         text: $t({defaultMessage: "Loading…"}),
     });
 
+    $("#uploaded_files_table").on("click", ".download-attachment", () => {
+        $("#hidden-attachment-download")[0]?.click();
```

Comment:
For some reason, wrong file is being downloaded. I uploaded three files, and same file is being downloaded while clicking on download button of all three files.

Reply:
Mostly to do with the `#hidden-attachment-download` id being repeated for every attachment, which we shouldn't be having. A better method would be to use `this` and track the closest element with the attachment.